tests/ui/entities/remote_entities/test_remote_entities_api.py

```diff
@@ -307,17 +307,17 @@ def test_sync_concept(
 
 
 def test_remote_entity_properties(
-    entity_person, item_lib_martigny, document, document_data
+    entity_person, item_lib_martigny, document, document_data, mef_concept1
 ):
     """Test entity properties."""
     item = item_lib_martigny
 
     assert document.pid not in entity_person.documents_pids()
-    assert document.id not in entity_person.documents_ids()
+    assert str(document.id) not in entity_person.documents_ids()
```

Comment:
You should also assert the index, here you are only testing the DB. Same below.

Reply:
`document_ids()` is a `DocumentsSearch`.